scania-digital-design-system/tegel
# test(spinner): added unit test

## **Describe pull-request**  
Adding unit test for Spinner component

## **Issue Linking:**  
- **Jira:** Add ticket number after `CDEP-`: [CDEP-3040](https://tegel.atlassian.net/browse/CDEP-3040)    

## **How to test**  
1. Open a branch to your local
2. Try to run `npm run test`
3. All tests should pass.
4. Try to change something in spinner test files, for example, size or variant
5. Test again, it should fail.

## **Checklist before submission**

## **Additional context**  
The last component to have unit tests ;) 


[CDEP-3040]: https://tegel.atlassian.net/browse/CDEP-3040?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
